TPO/src/main/java/org/uade/adt/ISet.java

```diff
@@ -9,7 +9,14 @@ public interface ISet {
      * Postcondicion: Agrega un valor al conjunto.
      * @param a valor a agregar.
      */
-    void add(int a);
+    ISet Add(int a);
+
+    /**
+     * Postcondicion: Agrega multiples valores al conjunto.
+     * @param set El set al cual vamos a agregar los valores
+     * @param values Los valores a ser agregados en el set
+     */
+    ISet AddAll(int[] values, ISet set);
```

Comment:
El addAll debería recibir únicamente el parámetro set. Elimina el parámetro values

Reply:
El addAll agregue a la instancia en la que estas parado (usando this) todos los valores que esten en el conjunto

Reply:
Actualizado